pages/vi/profiles/adewale2018.md

```diff
@@ -0,0 +1,32 @@
+#### `Firstname: ` SAHEED
+___
+#### `Middlename:` ADEWALE
+___
+#### `Surname:`  SHITTU
+___
+#### `Location:` LAGOS, NIGERIA
+___
+#### `Time zone:` GMT + 1
+___
+#### `OS:` WINDOWS 10, ACER 2018.
+___
+#### `ABOUT ME:`
+```
+    I am from Nigeria. I studied mechanical engineering and was working in a flour mill company before I switched
+    to learning software development. I always look for opportunity to acquire the technical and the soft skills 
+    needed to become a world-class software engineer. It gladdens my heart to be on the process of becoming a 
+    member of this great community. Nice to meet you all!.
+```
+___
+##### `JOURNEY OF ALMOST A YEAR:`
+ - HTML5
+  - CSS3
```

Comment:
I just noticed that indentation for the first element in the list is not the same as for other elements, and it creates a nested list. Could you please fix that.
![Screen Shot 2019-04-09 at 11 19 35 AM](https://user-images.githubusercontent.com/35533801/55824769-65825280-5ab9-11e9-9fa4-e091448f8bb5.png)